runtime-config.md

```diff
@@ -1,3 +1,14 @@
+# Runtime Configuration
+
+## Hooks
+
+Lifecycle of a container's runtime, for setup and teardown.
+Presently there are `Prestart` and `Poststop`.
+
+* `Prestart` is a list of hooks to be run before the container process is executed
+* `Poststop` is a list of hooks to be run after the container process exits
```

Comment:
The hooks aren't capitalized in the spec JSON, so I'd prefer `prestart` and `poststop`.  We're also not particularly consistent in how we markup JSON keys in the spec, but I think the recent trend has been to use **prestart** instead of `prestart`.